native/src/components/PoiFiltersModal.tsx

```diff
@@ -125,7 +127,7 @@ const PoiFiltersModal = ({
           </TileRow>
         </Section>
         <Section>
-          <StyledTextButton onPress={closeModal} text={t('showPois')} />
+          <StyledTextButton onPress={closeModal} text={`${poisLength} ${t('showPois')}`} />
```

Comment:
You should adjust our translations accordingly instead of using a template literal, i.e. adjusting the `showPois` translation such that you can pass a count and depending on that get the right singular/plural text. The reason for this is mainly that e.g. the word order might differ in other languages. As an example you can have a look at the docs: https://www.i18next.com/translation-function/plurals#languages-with-multiple-plurals

Reply:
Also, I think we should disable the button the same way as on web if there are no pois matching the filters.